### ISSUE TITLE
Build on windows 11

### ISSUE THREAD

**Issue by davidkhala** [davidkhala/databricks-common#3]
The downgrade `databricks-connect` (to 14.3.2) solution doesn't work since it has no serverless option in `DatabricksSession.builder`

**Comment by davidkhala** [comment]
workaround 
`pyproject.toml`
```
...
databricks-connect = "*" 
numpy= "^2" # fix issue up to databricks-connect:15.4.3 
setuptools= "*" # for `No module named 'distutils'`
```

**Comment by davidkhala** [comment]
It is resolved by downgrading python version from 3.13 to 3.12
